sites/mosul/configs/openmrs/initializer_config/ampathforms/F11-Family_Planning_Assessment.json

```diff
@@ -454,224 +462,82 @@
-              "label": "POP, Implant",
-              "type": "obs",
-              "required": false,
-              "questionOptions": {
-                "rendering": "text",
-                "concept": "popImplant"
-              },
-              "hide": {
-                "hideWhenExpression": "conditions !== 'pregnancy'"
-              }
-            },
-            {
-              "id": "iud",
-              "label": "IUD",
-              "type": "obs",
-              "required": false,
-              "questionOptions": {
-                "rendering": "text",
-                "concept": "iud"
-              },
-              "hide": {
-                "hideWhenExpression": "conditions !== 'pregnancy'"
-              }
-            },
-            {
-              "id": "injectable",
-              "label": "Injectable",
-              "type": "obs",
-              "required": false,
-              "questionOptions": {
-                "rendering": "text",
-                "concept": "injectable"
-              },
-              "hide": {
-                "hideWhenExpression": "conditions !== 'pregnancy'"
-              }
-            }
-          ]
-        },
-        {
-          "label": "Relative contra-indication:",
-          "isExpanded": false,
-          "questions": [
-            {
-              "id": "injectableImplant",
-              "label": "Injectable, implant",
-              "type": "obs",
-              "required": false,
-              "questionOptions": {
-                "rendering": "text",
-                "concept": "injectableImplant"
-              },
-              "hide": {
-                "hideWhenExpression": "conditions !== 'breastfeedingLessThan6Weeks'"
-              }
-            },
-            {
-              "id": "tubalLigation",
-              "label": "Tubal ligation",
-              "type": "obs",
-              "required": false,
-              "questionOptions": {
-                "rendering": "text",
-                "concept": "tubalLigation"
-              },
-              "hide": {
-                "hideWhenExpression": "conditions !== 'pregnancy'"
-              }
-            },
-            {
-              "id": "cocPop",
-              "label": "COC, POP",
-              "type": "obs",
-              "required": false,
-              "questionOptions": {
-                "rendering": "text",
-                "concept": "cocPop"
-              },
-              "hide": {
-                "hideWhenExpression": "conditions !== 'unexplainedVaginalBleeding'"
-              }
-            },
-            {
-              "id": "iud",
-              "label": "IUD",
-              "type": "obs",
-              "required": false,
-              "questionOptions": {
-                "rendering": "text",
-                "concept": "iud"
-              },
-              "hide": {
-                "hideWhenExpression": "conditions !== 'hyperOrDysmenorrhoea'"
-              }
```

Comment:
These questions are deleted, as they are listed in the metadata, but these are not questions, but mere helper texts to the selected options in the "Conditions" question, which will be done in [LIME2-586](https://msf-ocg.atlassian.net/browse/LIME2-586) and as confirmed [here](https://msf-ocg.atlassian.net/browse/LIME2-633?focusedCommentId=77857) by @michaelbontyes.
Thanks!